### PR TITLE
php-cs-fixer configuration

### PR DESCRIPTION
**Pull request by adamwojs**
# Description

This PR adds php-cs-fixer configuration and reformat code according to PSR-2 rules.

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?php
+
+if (class_exists('Symfony\CS\Config\Config')) {
+    // PHP-CS-Fixer 1.x syntax (deprecated)
```
Comment (andrerom) [inline .php_cs]: you don't need this, the reason why it is there on kernel was for the transition period between 1.x and 2.x. But we don't support using both, 1.x is by now dead.

Comment (andrerom) [reply .php_cs]: https://github.com/ezsystems/ezpublish-kernel/commit/b8af033907e7cdb70c388925da66c354e1a67dda

File: .php_cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ;
+}
+
+// PHP-CS-Fixer 2.x syntax
+return PhpCsFixer\Config::create()
+    ->setRules([
+        '@Symfony' => true,
+        '@Symfony:risky' => true,
+        'concat_space' => ['spacing' => 'one'],
+        'array_syntax' => false,
```
Comment (alongosz) [inline .php_cs]: @andrerom I know you are bored by this ;) but maybe here we can do `'array_syntax' => ['syntax' => 'short'],` and fix everything at once? :D
(composer requires at least PHP 5.4 and we don't have multiple supported branches yet).

File: .php_cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        PhpCsFixer\Finder::create()
+            ->in(__DIR__)
+            ->exclude([
+                'docs',
+                'vendor',
+                'src/bin'
+            ])
+            ->files()->name('*.php')
+    )
+;
```
Comment (alongosz) [inline .php_cs]: Please add new line at the end of the file.

**Review by alongosz — CHANGES_REQUESTED** [review]
General remark:
Please add `.php_cs.cache` to `.gitignore`

File: .php_cs (hunk 2 above)
Comment (andrerom) [reply .php_cs]: it's ok on this repo

**Comment by adamwojs** [comment]
@andrerom @alongosz Thank you for review, I've changed this PR according to your suggestions. Is there needed some extra configuration to run this by Travis on each PR?

**Review by alongosz — APPROVED** [review]

**Review by andrerom — APPROVED** [review]

**Pull request merged**